index.html

```diff
@@ -117,38 +117,20 @@ <h2>Network requests</h2>
 
       <p>
       A <dfn data-lt="network requests">network request</dfn> occurs when the
-      user agent must use the network to service a single
-      <dfn data-cite="RFC9110#core.semantics">request</dfn>.
+      [=user agent=] attempts to <a data-cite="fetch#http-network-fetch">HTTP-network fetch</a>
+      a resource over the network for a given [=request=].
       </p>
 
       <p>
-      If the user agent can service a <a>request</a> out of a local
-        cache, that <a>request</a> MUST NOT result in a <a>network
-        request</a>.
+      A [=request=] MUST NOT result in a <a>network request</a> if the user
+      agent is known to be offline (i.e., when <code>navigator.</code>
+      {{NavigatorOnLine/onLine}} returns <code>false</code>).
       </p>
 
       <p>
-      If the user agent follows [=redirect status|redirects=] as part of a [=navigate|navigation=],
-      there MUST be separate <a>network requests</a> for each <a>request</a> in
-      the redirect chain.
-      </p>
-
-      <p>
-      A <a>request</a> MUST NOT result in a <a>network request</a> if the user

```

Comment:
These two paragraps likely follow from fetch, but may be useful as a note

Reply:
Thanks, will swap to a Note.

Reply:
I think we're covered - the navigator.onLine requirement is just moved above, and the CORS failure requirement is in tact. I'll add back the note that was deleted, because it's non-normative anyways and we can delete it later.